README.md

```diff
@@ -1,6 +1,6 @@
 # gauges2
```

Comment:
Should we rename the header as well?

Reply:
https://github.com/ToyotaResearchInstitute/gauges2/pull/23/commits/e22589c36778147fc4e9b1c676ab864a65b58de2